### PR TITLE
Create outline for cebu philippines tourism information

### LINKED ISSUE #80: n0rbs Tr@ck Progr3ss on Github Work Flow
# n0rbs workflow 🐶
- [x] Create a branch
- [x] add commits on my branch
- [x] open a PR
- [x] collaborate and have a conversation
- [x] make some more commits
- [x] merge our branch with master

> (' ' , )

### PR DESCRIPTION
**Pull request by n0rbs**
This `PR` adds forlder _norbs_ and file _norbs-cebu-singapore.md_

@brianamarie Please visit the Philippines 👍 🐟 🎆 

This `PR` resolves #80 🐶

### CODE REVIEW

File: norbs/n0rbs-cebu-singapore.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Cebu Philippines
+
+### Food Places
```
Comment (n0rbs) [inline norbs/n0rbs-cebu-singapore.md:3]: Reduce the size of this text

**Comment by githubteacher** [comment]
@n0rbs I'd love to visit the Philippines! 🐟

File: norbs/n0rbs-cebu-singapore.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Cebu Philippines
```
Comment (david-w) [inline norbs/n0rbs-cebu-singapore.md:1]: @n0rbs Philippines? What time zone is that?

Comment (n0rbs) [inline norbs/n0rbs-cebu-singapore.md:1]: @david-w +08:00 📱 🐟 🐮 🐶 warm weather

**Comment by gitalinoe** [comment]
love Cebu

**Pull request merged**